LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2018 Brainbot Labs Est.
```

Comment:
Lower case -> [https://brainbot.li/](https://brainbot.li/)
```suggestion
Copyright (c) 2015-2018 brainbot labs Est.
```

Reply:
We use the upper case spelling in the raiden repo license. If this is the correct one we should change it there as well